src/components/CreateProjectDialog/CreateProjectDialog.tsx

```diff
@@ -0,0 +1,125 @@
+import * as Dialog from '@radix-ui/react-dialog';
```

Comment:
If this is only used on the dashboard, I wonder if we should just move it into `apps/dashboard-projects/CreateProjectDialog`?

Reply:
Yeah.  I have not really always made the Only used in this app, general purpose component.  I am not sure we have been totally consistent on this.